openshift/templates/backup/build.yaml

```diff
@@ -2,19 +2,19 @@
 kind: Template
```

Comment:
We can't currently build the mssql back as the PR was never merged with the original repo.